app/scripts/modules/core/src/presentation/forms/Validation.ts

```diff
@@ -0,0 +1,11 @@
+export type ValidationFunction = (value: any) => string | Function | Promise<any>;
+
+export class Validation {
```

Comment:
I originally had individual exports, but switched to static properties because:

- it seemed more explicit to write `Validation.isRequired` than `isRequired`.
- it would force us to be more consistent (not switching between `isRequired` and `Validation.isRequired` arbitrarily).  
- it would enforce the variable name for the latter style (no change of typing `import * as Validators from 'core/presentation'`)
- it allows you to discover validations by typing `Validation.<ctrl-space>`

At any rate, feel free to argue back if you prefer it.

Reply:
I definitely agree with the point on enforcing import naming. I was surprised to see the thing about autocomplete though — I use ctrl + space autocomplete for that import style frequently in VS Code 🤭